kafka-ui-api/src/main/java/com/provectus/kafka/ui/exception/SchemaTypeIsNotSupportedException.java

```diff
@@ -0,0 +1,8 @@
+package com.provectus.kafka.ui.exception;
+
+public class SchemaTypeIsNotSupportedException extends UnprocessableEntityException {
+
+  public SchemaTypeIsNotSupportedException() {
+    super("Current version of Schema Registry does not support provided schema type");
```

Comment:
Could we state the minimum required version of SR which supports schemaType here please?

Reply:
Updated exception message with the minimum required version of SR. @Haarolean  Could you check it, pls?

Reply:
Nice, thanks